collect_app/src/test/java/org/odk/collect/android/formentry/audit/FormSaveViewModelTest.java

```diff
@@ -68,38 +68,44 @@ public void setup() {
 
     @Test
     public void saveForm_returnsNewSaveResult_inSavingState() {
-        LiveData<FormSaveViewModel.SaveResult> saveResult1 = viewModel.saveForm(Uri.parse("file://form"), true, "", false);
```

Comment:
Yeah I think this test also needs to change as the `LiveData` will be the same between calls. Should just be that `saveForm_returnsSaveResult_inSavingState` and it only needs to make one call to `save`.

Reply:
Done.